src/tileset/wangset/wang_tile.rs

```diff
@@ -0,0 +1,64 @@
+use std::str::FromStr;
+
+use xml::attribute::OwnedAttribute;
+
+use crate::{
+    error::Error,
+    util::{get_attrs, XmlEventResult},
+    Result, TileId,
+};
+
+/**
+The Wang ID, stored as an array of 8 u32 values.
+*/
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub struct WangId(pub [u32; 8]);
```

Comment:
I wonder a bit why `u32` is used here instead of `u8`. In Tiled, a `WangId` is actually stored as a single `u64`, but storing as 8 times `u8` would also make sense. It's just that some operations, like masking, are easier to do on a single value than on an array of bytes.

Reply:
You are right, I see now in the docs that this field should be u8 values instead of u32.

I was mistaken because this was not specified on TMX format docs, but ti is in JSON

> wangid: "The Wang ID, given by a comma-separated list of indexes (starting from 1, because 0 means _unset_) referring to the Wang colors in the Wang set in the following order: top, top right, right, bottom right, bottom, bottom left, left, top left (since Tiled 1.5). Before Tiled 1.5, the Wang ID was saved as a 32-bit unsigned integer stored in the format 0xCECECECE (where each C is a corner color and each E is an edge color, in reverse order)."

Reply:
Thanks for pointing that out, I'll push a small update to those docs to mention the range of the indexes (0-255).